src/NLU.DevOps.Luis.Shared/LuisNLUTrainClient.cs

```diff
@@ -35,13 +34,28 @@ public LuisNLUTrainClient(ILuisConfiguration luisConfiguration, LuisSettings lui
             this.LuisSettings = luisSettings ?? throw new ArgumentNullException(nameof(luisSettings));
             this.LuisClient = luisClient ?? throw new ArgumentNullException(nameof(luisClient));
             this.LuisAppId = luisConfiguration.AppId;
+            this.LuisAppCreated = luisConfiguration.AppCreated;
         }
 
         /// <summary>
         /// Gets the LUIS app ID.
         /// </summary>
         public string LuisAppId { get; private set; }
 
+        /// <summary>
+        /// Gets the LUIS version ID.
+        /// </summary>
+        public string LuisVersionId => this.LuisConfiguration.VersionId;
```

Comment:
Delete, not needed, the version ID is produced deterministically from the configuration.